frontend/src/js/components/TicketCountSelector.jsx

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+var _ = require('underscore');
+var React = require('react');
+var Seat = require('./Seat.jsx');
+
+var Input = require('react-bootstrap/lib/Input');
+
+var TicketCountSelector = React.createClass({
+
+  getInitialState: function() {
+    return {error: null};
+  },
+
+  onChange: function(section, event) {
+    var newVal = event.target.value;
+    var sectionSeatIds = _.values(_.mapObject(this.props.venue.get('sections')[section.id].seats, function(seat) { return seat.id;})); // _.keys returns strings, we need ints
+    var sectionChosenSeatIds = _.intersection(this.props.chosenSeatIds, sectionSeatIds);
+    var current = sectionChosenSeatIds.length;
+    if (current > newVal) {
+      if (newVal < 0) {
```

Comment:
wonder if we should have an upper bound. On one hand, it's trivial to get around such a limitation (another window in incognito mode), on the other, it would prevent accidental "oops, I reserved 100 tickets instead of 10 and locked the whole show from other people for 15min". Thoughts?